scripts/ci/deploy/databend-query-sharing.sh

```diff
@@ -31,3 +31,12 @@ python3 scripts/ci/wait_tcp.py --timeout 10 --port 23003
 echo "Start query node2 for sharding data"
 nohup target/${BUILD_PROFILE}/databend-query -c scripts/ci/deploy/config/databend-query-node-share-2.toml &
 python3 scripts/ci/wait_tcp.py --timeout 30 --port 13307
+
+echo 'Start query node3 open-sharing...'
+nohup target/${BUILD_PROFILE}/open-sharing --tenant=shared_tenant --storage-type=s3 --storage-s3-bucket=testbucket --storage-s3-endpoint-url=http://127.0.0.1:9900 --storage-s3-access-key-id=minioadmin --storage-s3-secret-access-key=minioadmin --storage-allow-insecure --share-endpoint-address=127.0.0.1:23103 &
+python3 scripts/ci/wait_tcp.py --timeout 10 --port 23103
+
+echo "Start query node3 with the same tenant and meta server for sharding data"
+nohup target/${BUILD_PROFILE}/databend-query -c scripts/ci/deploy/config/databend-query-node-share-3.toml &
```

Comment:
What will happen if a request is sent to query node 1 to access the sharing table?

Reply:
Now cannot get `inbound` shared tables without `open-sharding` service

Reply:
I'm a bit confusing about the config value. Do we need to keep config value of `sharing_endpoint_address` the same across the same cluster?